### DISCUSSION TITLE
clarification re. replace cloudflare/cloudflare-ipv6 with 'odoh-cloudflare' - both entries or just one?

### DISCUSSION THREAD

**Discussion by mgrimace** [trinib/Adguard-Wireguard-Unbound-Cloudflare#75] [Help]
Hello,
I'd like to use ODOH, but it isn't clear to me if the `/` is an 'and' or an 'or', for example should it be:

server_names = ['odoh-cloudflare', 'odoh-cloudflare'], or server_names = ['odoh-cloudflare'], or something else.
Thanks for the quick clarification.

**Comment by trinib** [comment]
i see what you mean . Remove both server names 'cloudflare' and 'cloudflare-ipv6' (which is two string elements in a list in code syntax)
```
server_names = ['cloudflare', 'cloudflare-ipv6']
```
And add 'odoh-cloudflare' string ONCE in list of elements, not twice 
```
server_names = ['odoh-cloudflare']
```

I will try and clarify in guide
